addons/search-sprites/search-bar.css

```diff
@@ -0,0 +1,15 @@
+#sa-sprite-search-box {
+    color: black;
+    width: 100%;
+    box-sizing: border-box;
+    background-color: #eaf1fb;
+    border: 0px;
+    border-bottom: 1px solid hsla(0, 0%, 1%, 0.15);
+    outline: none;
+    padding: 5px 10px;
+}
+
+input[type="search"]::-webkit-search-cancel-button,
+input[type="search"]::-webkit-search-decoration {
+    -webkit-appearance: auto;
```

Comment:
This only works in chromium.

Reply:
That exists as a flag in Firefox. `layout.forms.input-type-search.enabled` is the name of the flag.
<img width="348" alt="firefox" src="https://user-images.githubusercontent.com/83728060/147356101-3c1ba080-7362-4dbc-ad2a-d381d97ca0bf.png">

Reply:
we don't want them to have to use a flag